corehq/apps/app_manager/views/app_summary.py

```diff
@@ -142,7 +143,7 @@ def _translate_name(names, language):
     try:
         return names[language]
     except KeyError:
-        first_name = names.iteritems().next()
+        first_name = six.iteritems(names)
```

Comment:
this looks different than what was there before. i think you are missing `.next()`

Reply:
Good catch - updated